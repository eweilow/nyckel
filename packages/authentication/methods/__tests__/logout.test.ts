import { authorizeUser, GlobalAuthenticationConfig, logoutUser } from "../..";

const config: GlobalAuthenticationConfig = {
  audience: "param:audience",
  authorizationDomain: "http://param:authorizationDomain.com",
  clientId: "param:clientId",
  clientSecret: "param:clientSecret",
  urls: {
    authorization: "url:authorization",
    issuer: "url:issuer",
    jwks: "url:jwks",
    logout: "url:logout",
    token: "url:token",
    userinfo: "url:userinfo"
  }
};

describe("logoutUser", () => {
  it("returns correctly", async () => {
    const data = await logoutUser("redirectUri", config);
    expect(data).toMatchSnapshot();
  });
});
