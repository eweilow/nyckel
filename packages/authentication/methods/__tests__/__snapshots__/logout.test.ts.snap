// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logoutUser returns correctly 1`] = `
Object {
  "redirectTo": "url:logout?client_id=param%3AclientId&returnTo=redirectUri",
}
`;
